package.json

```diff
@@ -29,6 +29,8 @@
   ],
   "devDependencies": {
     "babel-preset-es2015": "^6.18.0",
+    "eslint-config-incuna": "1.0.0",
+    "eslint-plugin-import": "2.2.0",
```

Comment:
But it might backfire if there is a need for just a small change in client copy for example, which can be made from github even by a PM and suddenly someone else has to jump in and fix the linting. I'm against this idea.